packages/react-dom/src/shared/warnValidStyle.js

```diff
@@ -16,7 +16,6 @@ var warnValidStyle = emptyFunction;
 if (__DEV__) {
   var camelizeStyleName = require('fbjs/lib/camelizeStyleName');
   var warning = require('fbjs/lib/warning');
-  var {getCurrentFiberOwnerName} = require('ReactDebugCurrentFiber');
```

Comment:
Removing dependency on Fiber-only module from something that can be used on the server.